core/src/main/web/app/mainapp/components/pluginmanager/pluginmanager-directive.js

```diff
@@ -145,5 +145,23 @@
       }
     };
 
+    $rootScope.$on(GLOBALS.EVENTS.LANGUAGE_MANAGER_SHOW_SPINNER, function(event, data) {
+      $scope.showSpinner = true;
+      $scope.showMessage = true;
+      $scope.loadingMessage = 'Restarting ' + data.pluginName + '...';
+    });
+
+    $rootScope.$on(GLOBALS.EVENTS.LANGUAGE_MANAGER_HIDE_SPINNER, function(event, data) {
+      if (data.error) {
+        $scope.loadingMessage += 'failed';
+      } else {
+        $scope.loadingMessage += 'done';
+      }
+      $scope.showSpinner = false;
+      bkUtils.timeout(function() {
+        $scope.showMessage = false;
+      }, 3000);
```

Comment:
3s is supposed to be how long it takes for the message to fade out after the reset is finished.